CHANGELOG.md

```diff
@@ -56,6 +56,7 @@
 - dev: add avail and celestia crates behind a feature flag
 - dev: replace md5 with sha3_256 hash function
 - feat: fixing getNonce Rpc Call and adding a new test
+- dev: fix linter warning in README.md
```

Comment:
can you add this to the top of Next Release

Reply:
done